src/main/java/entity/Attachment.java

```diff
@@ -45,7 +45,7 @@ public Attachment(long attachmentId) {
     public Attachment(long attachmentId, MultipartFile file) {
         this.attachmentId = attachmentId;
         this.fileName = file.getOriginalFilename();
-        this.fileType = file.getContentType();
+        this.fileType = fileName.split("\\.")[1];
```

Comment:
not all files have extensions.
Please add suitable handling for that case here.
Perhaps we can store an empty string there to signify an unknown type.